### DISCUSSION TITLE
How to compare nested polr models?

### DISCUSSION THREAD

**Discussion by trelse** [amices/mice#369] [Q&A]
Hello,

I saw that D1 is not written for polr model from the MASS package. 
Would you recommend to still use pool.compare(method = "wald") with the Dm test statistics for this comparison? Or what would you recommend?
Thanks a lot.

**Comment by stefvanbuuren** [comment]
I recommend `D1()` and `D3()`. Use `mice 3.8.2`. See <https://stefvanbuuren.name/fimd/sec-multiparameter.html#d_1-d_2-or-d_3> for details.

`D2()` is methodologically inferior, and `pool.compare()` is outdated.
